src/gl.rs

```diff
@@ -380,6 +380,7 @@ pub trait Gl {
     fn get_buffer_parameter_iv(&self, target: GLuint, pname: GLenum) -> GLint;
     fn get_shader_info_log(&self, shader: GLuint) -> String;
     fn get_string(&self, which: GLenum) -> String;
+    fn get_string_i(&self, which: GLenum, index: GLuint) -> String;
```

Comment:
@glennw I wonder if we'd be better having something like `get_string(&self, which: GLenum, index: Option<GLuint>)`, since this is essentially what GL API does here